src/VoteDelegate.sol

```diff
@@ -95,6 +104,11 @@ contract VoteDelegate {
         emit Free(msg.sender, wad);
     }
 
+    function reserveHatch() external {
+        require(block.number > hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
```

Comment:
Since liquidators will be using this and they are often very gas-sensitive, it would be nice (but optional) to make the math here `unchecked` as realistically there should never be overflow.

Reply:
Yeah, I think we'll leave it as is. Usually using the `unchekced` scoping only when the calculation is separate.